designsafe/apps/api/licenses/views.py

```diff
@@ -0,0 +1,26 @@
+from designsafe.apps.api.views import BaseApiView
+from designsafe.apps.api.mixins import SecureMixin
+from designsafe.libs.common.decorators import profile as profile_fn
+from django.contrib.auth import get_user_model
+from django.http.response import HttpResponseForbidden, HttpResponseNotFound
+from django.http import JsonResponse
+from django.apps import apps
+from designsafe.apps.data.models.agave.util import AgaveJSONEncoder
+
+
+class LicenseView(SecureMixin, BaseApiView):
+    @profile_fn
+    def get(self, request, app_name, username):
+        if not request.user.is_staff:
+            return HttpResponseForbidden()
+        
+        try:
+            app_license = apps.get_model('designsafe_licenses', '{}License'.format(app_name))
+        except:
```

Comment:
maybe we can try this error specifically
```suggestion
        except LookupError:
```